### ISSUE TITLE
Matlab RANDOMSEARCH is no uniform sampling

### ISSUE THREAD

**Issue by brockho** [numbbo/coco#705]
The currently implemented random search in the Matlab/Octave example experiment is not sampling uniformly at random in between the upper and lower bounds:
```
function my_optimizer (f, lower_bounds, upper_bounds, budget)
    n = length(lower_bounds);
    delta = upper_bounds - lower_bounds;
    for i= 1:budget
        x = lower_bounds + normrnd(zeros(1, n), 1) .* delta;
        y = cocoCall('cocoEvaluateFunction', f, x);
    end
end
```
I would call this at least a semi-bug.

**Comment by nikohansen** [comment]
Given the algorithm doesn't consider itself as uniform sampling, it would give it a pass as a slight misnomer (lower domain of interest value is named as lower bound).

**Comment by brockho** [comment]
Just an addition: The `exampleexperiment.m` currently starts with the following comment:
```
% This script runs random search for BUDGET_MULTIPLIER*DIM function
% evaluations on the biobjective 'bbob-biobj' suite.
```

**Comment by nikohansen** [comment]
Agreed in that the main problem is that reported results are likely to report a wrong underlying algorithm.

**Comment by brockho** [comment]
solved by commit 72f0a0c
